src/components/Home.jsx

```diff
@@ -49,24 +48,7 @@ class Home extends Component {
             <Services />
           </div>
         </Content>
-        <Switch>
-          <Route
-            exact
-            path="/connected/:konnectorSlug"
-            component={AccountPicker}
-          />
-          <Route
-            path="/connected/:konnectorSlug/new"
-            render={props => (
-              <ConnectionManagement originPath="/connected" {...props} />
-            )}
-          />
-          <Route
-            path="/connected/:konnectorSlug/accounts/:accountId"
-            component={AccountPicker}
-          />
-          <Redirect from="/connected/*" to="/connected" />
```

Comment:
Shouldn't we keep this one around?

Reply:
You right, this way we can redirect if one of the previous Route was not hit `connected/foo/bar/test` for instance.